diff --git a/node_modules/@vscode/spdlog/binding.gyp b/node_modules/@vscode/spdlog/binding.gyp
index 5daaca4..7c591f6 100644
--- a/node_modules/@vscode/spdlog/binding.gyp
+++ b/node_modules/@vscode/spdlog/binding.gyp
@@ -16,9 +16,6 @@
 			"NODE_API_SWALLOW_UNTHROWABLE_EXCEPTIONS",
 			"FMT_CONSTEVAL="
 		],
-		'msvs_configuration_attributes': {
-			'SpectreMitigation': 'Spectre'
-		},
 		'msvs_settings': {
 			'VCCLCompilerTool': {
 				'ExceptionHandling': 1,
